<!DOCTYPE html>
<html lang="en"><head>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/tabby.min.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.7.32">

  <title>talks – ctDNA in Esophageal Cancer</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="site_libs/revealjs/dist/theme/quarto-f563837468303362081e247dddd440d0.css">
  <link href="site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">ctDNA in Esophageal Cancer</h1>

<div class="quarto-title-authors">
</div>

</section>
<section id="ctdna-in-esophageal-cancer" class="slide level2">
<h2>ctDNA in Esophageal Cancer</h2>
<p>Identification of patients at elevated risk of systemic relapse is critical in the choice of optimal therapy at multiple points in the treatment pathway.</p>
<p>Neoadjuvant chemoradiation has been demonstrated to be a useful adjunct to surgery in locally-advanced esophageal cancer without radiographic evidence of distant metastasis. Long-term analysis of the CROSS trial demonstrated that the addition of chemoradiation to surgery primarily reduces the risk of locoregional and mixed locoregional and systemic recurrences, but does not reduce the risk of isolated systemic recurrence.</p>
<p>There is clearly heterogeneity of tumor sensitivity to chemoradiation, as borne out by the observation from the CROSS trial of trimodality therapy, in which a pathologic complete response was noted in 23% of patients with adenocarcinoma and 49% of patients with squamous cell carcinoma. At the other end of spectrum, 30% of patients treated with trimodality therapy experienced systemic failure by 2 year after completion of treatment.</p>
<p>The TOPGEAR study was designed to reduce the risk of systemic recurrence by the use of induction chemotherapy prior to chemoradiation. The study did not show a difference in overall survival.</p>
<p>An understanding of the risk of systemic recurrence is crucial for rational decision-making regarding neoadjuvant therapy. Patients at low risk of systemic failure may be best treated with trimodality therapy along the lines of the CROSS trial, while patients at elevated risk of systemic failure may benefit from induction chemotherapy followed by chemoradiation.</p>
<p>For patients who have completed neoadjuvant therapy and had no evidence of metastatic disease on restaging, optimal therapy is to proceed to surgery. The ideal candidates for surgical therapy are those who have persistent local disease but have a low risk of occult metastatic disease.</p>
<p>For patients who have clinical evidence of persistent local disease after neoadjuvant therapy, the treatment options are surgery or systemic therapy. In this scenario, the risk of occult metastatic disease would be helpful in treatment decisions.</p>
<p>For patients who have had a complete clinical response, as judged by negative endoscopic biopsies and a robust response on imageing, active surveillance is an option. Active surveillance allows the natural history of esophageal cancer to become manifest during a period of regular endoscopy and imaging.</p>
<p>The SANO clinical trial (Surgery As Necessary Only) focused on patients with esophageal cancer who had completed chemoradiation and had no evidence of persistent disease on endscopy or imaging. They were randomized to immediate surgery or active surveillance. Active surveillance entailed periodic endoscopy and imaging. Patients found to have an isolated local recurrence were recommended for salvage esophagectomy. The study was designed as a non-inferiority trial with an acceptable survival difference between arms of 15%. Active surveillance was found to be non-inferior to immediate surgery.</p>
<p>For patients with a complete clinical response, the treatment options are immediate surgery, systemic chemotherapy, or active surveillance. These choices are again informed by an understanding of the risk of occult metastatic disease. Patients at high risk for systemic disease are likely best treated with systemic chemotherapy, while patients at low risk of systemic disease may ideal candidates for active surveillance, as a number of these patients may be found to have eradication of their disease by chemoradiation and may never need surgery.</p>
<p>Circulating tumor DNA (ctDNA) represents a small fractino of circulating cell-free ctDNA or “liquid biopsy” is a promising technology to estimate the risk of systemic disease at multiple points in a patient’s treatment pathway.</p>
<p>The presence of circulating, tumor DNA (ctDNA) in plasma is well established(11–13). In most patients, ctDNA represents a small fraction of the total circulating cell-free DNA (ccfDNA)(14), and is challenging to detect and quantify. Novel technologies, such as our sequencing approach(15,16), enable detection of these rare tumor mutations in an abundance of normal ccfDNA. Several studies have indicated that ctDNA burden correlates closely with overall disease burden(11,14,17,18). Thus, ctDNA provides a potential “liquid biopsy” that can be used to monitor cancer progression and recurrence during treatment.</p>
<ul>
<li><p>Initial diagnosis</p></li>
<li><p>MRD window</p></li>
<li><p>Surveillance</p></li>
</ul>

<img data-src="images/defize.png" class="r-stretch quarto-figure-center"><p class="caption">Defize Algorithm</p><p><strong>Rotterdam</strong> <span class="citation" data-cites="eyck35">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Eyck et al., 2023</a>)</span> Detection of circulating tumour DNA after neoadjuvant chemoradiotherapy in patients with locally advanced oesophageal cancer. J Pathol 2023; 259: 35–45. Samples taking before, 6 weeks after and 23 weeks after CRT. 29/31 patients had mutations detected on <em>tissue biopsy</em>. 24/31 (77%) had mutations potentially detectable in cfDNA. Pretreatment ctDNA detected in 9 (9/24 = 38%( patients, four of whom had incurable disease progression before surgery (2 prior to ChemoRT and 2 prior to surgery). None of these 9 had a pCR. 6/9 had major residual disease after chemoRT. After ChemoRT, ctDNA was detected in 3 patients (3/24 = 12.5%), two of whom had disease progression. Patients were examined wtih CT and EGD biopsies after ChemoRT as part of the SANO trial. ctDNA did not detect any additional patients with residual disease after ChemoRT. ctDNA analyzed with <a href="https://www.thermofisher.com/us/en/home/life-science/cancer-research/cancer-genomics/liquid-biopsy-cancer-research-applications/oncomine-cell-free-dna-assays-liquid-biopsy-clinical-research.html">Thermo Fisher Colon Oncoline kit</a>. <span class="math inline">\(\Rightarrow\)</span>. ctDNA can detect some patients with high-risk disease who are at risk for progression (and should have further staging investigations prior to surgery OR be offered watch/wait), but did not help detect patients after ChemoRT who had residual disease (who would be candidates for salvage esophagectomy)</p>
<p>Melbourne <span class="citation" data-cites="cabalage120">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Cabalag et al., 2022</a>)</span> 62 patients with adenocarcinoma with <strong>tumor-informed</strong> approach. 9 genes analyzed with targeted amplicon sequencing. Pre-treatment biopsies showed mutations in 55/62 (63%). ctDNA at baseline in 20/55 (36%). The majority (90%) of patients with detectable ctDNA had either locally advanced tumors, nodal involvement or metastatic disease. In patients with locally advanced tumors, disease free survival (DFS) was more accurately stratified using pre-treatment ctDNA status [HR 4.34 (95% CI 0.93-20.21); P = 0.05] compared to nodal status on PET-CT. In an exploratory subgroup analysis, patients who are node negative but ctDNA positive have inferior DFS [HR 11.71 (95% CI 1.16-118.80) P = 0.04]. In blood samples taken before and following NAT, clearance of ctDNA after NAT was associated with a favourable response to treatment. Furthermore, patients who are ctDNA positive during post-treatment surveillance are at high risk of relapse.</p>
<p>San Diego: <span class="citation" data-cites="kato6248">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Kato et al., 2018</a>)</span> 55 GE adenocarcinoma (mostly advanced disease; 9, surgically resectable) ctDNA with Guardant (49/55) or Foundation. Esophagus =11, GEJ = 17, gastric = 27. Seventy-six percent of patients (42/55) had ≥1 genomic alteration [including variants of unknown significance (VUS)] and 69.1% (38/55) had ≥1 characterized alteration (excluding VUSs). The median number of alterations per patient was 2 (range, 0–15). TP53 (50.9%, 28/55), PIK3CA (16.4%, 9/55), ERBB2 (14.5%, 8/55), and KRAS (14.5%, 8/55) genes were most frequently affected characterized alterations. Thirty-one patients also had tissue NGS. Concordance between tissue and ctDNA ranged from 61.3% (TP53 alterations) to 87.1% (KRAS alterations). ERBB2 alterations were significantly associated with poor overall survival (HR, 14.06; 95% confidence interval, 2.44–81.03; P = 0.003 multivariate analysis). Among patients with ≥1 alteration, no 2 patients had identical molecular portfolios.</p>
<p>Stanford: <span class="citation" data-cites="azad494">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Azad et al., 2020</a>)</span>: 45 patients receving chemoradiation for esophageal cancer. ctDNA analysis with deep squencing. Tumor-derived DNA represented 0.07% of all cell-free DNA. Detection of ctDNA following chemoradiotherapy was associated with tumor progression (hazard ratio, 18.7; P&lt;.0001), formation of distant metastases (hazard ratio, 32.1; P&lt;.0001), and shorter disease-specific survival times (hazard ratio, 23.1; P&lt;. 0001). A higher proportion of patients with tumor progression had new mutations detected in plasma samples collected after chemoradiotherapy than patients without progression (P=.03). Detection of ctDNA after chemoradiotherapy preceded radiographic evidence of tumor progression by an average of 2.8 months. Among patients who received chemoradiotherapy without surgery, combined ctDNA and metabolic imaging analysis predicted progression in 100% of patients with tumor progression, compared with 71% for only ctDNA detection and 57% for only metabolic imaging analysis. Before ChemoRT, ctDNA detected in 27/45 = 60%. After ChemoRT, ctDNA detected in 5/31 = 16%.</p>
<p>Zaanan A, Didelot A, Broudin C, et al.&nbsp;Longitudinal circulating tumor DNA (ctDNA) analysis during treatment (Tx) of locally advanced resectable (LAR) gastric or gastroesophageal junction (G/GEJ) adenocarcinoma (ADENOCA): the PLAGAST prospective biomarker study. J Clin Oncol 2024; 42: 4028 (abstract).</p>
<p>ctDNA in patients with pCR or near-pCR after chemoRT + surgery for esophageal cancer <span class="citation" data-cites="landere2400288">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Lander et al., 2024</a>)</span>. 42 patients (309 samples) taken during 16-week MRD interval and during surveillance and analyzed with Signatera. During MRD interval, 23 patients had sampled drawn during the MRD window, of whom 3 had detectable ctDNA with 2/3 recurrences compared with 3/20 (15% among those with negative ctDNA. ctDNA detected during MRD window was associated with shorter disease-free survival (HR 6.2 p=0.049). 32 patients were studied during surveillance, of whom 5/5 (100%) recurred in those with positive ctDNA and 2/27 (7.4%) among ct-DNA negative.</p>
<p>CRITICS trial <span class="citation" data-cites="leal525">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick=""><strong>leal525?</strong></a>)</span> identified post-neoadjuvant therapy or preoperative ctDNA as a predictive biomarker for pathological response (according to Mandard’s tumour regression grading [TRG] system0, with ctDNA analyses being consistent with TRG in 30 (70%) of 43 patients (p=0·03; Fisher’s exact test).</p>
<p>In the initial analysis of the PLAGAST study postoperative, tumour-informed ctDNA persistence was related to recurrence and overall survival. Additionally, the clearance of ctDNA after medical treatment could serve as a predictor of outcomes.</p>
<p>in the metastatic setting, Kelly and colleagues observed in a phase 1–2 study a statistical correlation between mutation variant allele frequencies and overall response rate to ICIs in metastatic oesophagogastric adenocarcinoma. Patients with a decrease in variant allele frequencies at week 9 had a longer median OS (3.8 months, 95% CI 7·0–NR) than those with an increase in variant allele frequencies (8.2 months, 6·2–NR; HR 0·07, 95% CI 0·01–0·66).74 However, false-negative ctDNA results have been reported in patients with peritoneal carcinomatosis, restricting the utility of ctDNA as the sole predictive biomarker for oesophagogastric adenocarcinoma.</p>
<p><strong>Rochester and Pittsburgh</strong> <span class="citation" data-cites="egyud343">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Egyud et al., 2019</a>)</span> Tumor-informed detection of ctDNA within patients with esophageal adenocarcinoma. Percentage of patients with detectable ctDNA increased with tumor stage. For patients with stage I or II disease, ctDNA was present in less than 50%, limiting its ability to detect esophageal cancer at an early stage.</p>
<p>Chidambaram S, Markar SR. Clinical utility and applicability of circulating tumor DNA testing in esophageal cancer: a systematic review and meta-analysis. Dis Esophagus. 2022;35:doab046.</p>
<p><span class="citation" data-cites="huffmane2200420">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Huffman et al., 2022</a>)</span> ctDNA levels were analyzed in the preoperative, postoperative, and surveillance settings in patients with EGC using a <strong>personalized multiplex polymerase chain reaction-based next-generation sequencing assay</strong>. Plasma samples (n = 943) from 295 patients at &gt; 70 institutions were collected before surgery, postoperatively, and/or serially during routine clinical follow-up from September 19, 2019, to February 21, 2022. A total of 295 patients with EGC were analyzed, and 212 patients with stages I-III disease were further explored. Pretreatment ctDNA was detected in 96% (23/24) of patients with preoperative time points. Postoperative ctDNA was detected in 23.5% (16/68) of patients with stage I-III EGC within 16 weeks (molecular residual disease window) after surgery without receiving systemic therapy. ctDNA detection at any time point after surgery (hazard ratio [HR], 23.6; 95% CI, 10.2 to 66.0;&nbsp;<em>P</em>&nbsp;&lt; .0001), within the molecular residual disease window (HR, 10.7; 95% CI, 4.3 to 29.3;&nbsp;<em>P</em>&nbsp;&lt; .0001), and during the surveillance period (HR, 17.7; 95% CI, 7.3 to 50.7;&nbsp;<em>P</em>&nbsp;&lt; .0001) was associated with shorter recurrence-free survival. In multivariable analysis, ctDNA status and clinical stage of disease were independently associated with outcomes. 125 patients with stage I–III oesophagogastric cancers were analysed at any timepoint postoperatively (regardless of adjuvant treatment). The median recurrence-free survival was not reached in patients negative for ctDNA, whereas it was 9·6 months in patients positive for ctDNA (HR 23·6, 95% CI 10·2–66·0; p=0·0001). These results were observed irrespective of the anatomical localisation of the tumour. Recurrence rates were 5% in patients negative for ctDNA and 88% in patients positive for ctDNA. The patient population enrolled in this study was heterogeneous and further prospective studies are required.</p>
<p><strong>Copenhagen</strong> <span class="citation" data-cites="iden83">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Iden et al., 2025</a>)</span>: Patients with resectable gastric and GEJ AC, undergoing perioperative chemotherapy and surgery, were prospectively enrolled. Serial plasma samples were collected at baseline, after one cycle of chemotherapy, after preoperative chemotherapy, and after surgery. ctDNA was assessed by a <strong>ddPCR test (TriMeth),</strong> which targets the gastrointestinal cancer-specific methylation patterns of the genes C9orf50, KCNQ5, and CLIP4.</p>
<p><strong>Results:</strong> ctDNA analysis was performed on 229 plasma samples from 86 patients. At baseline, ctDNA was detected in 56% of patients, which decreased to 37% following one cycle of chemotherapy, 25% after preoperative chemotherapy and 15% after surgical resection. The presence of ctDNA after one cycle of chemotherapy was associated with reduced recurrence-free survival (RFS) (HR = 2.54, 95% confidence interval (CI) 1.33-4.85, p = 0.005) and overall survival (OS) (HR = 2.23, 95% CI 1.07-4.62, p = 0.032). Similarly, ctDNA after surgery was associated with significantly shorter RFS (HR = 6.22, 95% CI 2.39-16.2, p &lt; 0.001) and OS (HR = 6.37, 95% CI 2.10-19.3, p = 0.001). Multivariable regression analysis confirmed ctDNA after surgery as an independent prognostic factor (p &lt; 0.001).</p>
<p>Wo JY, Clark JW, Eyler CE, Mino-Kenudson M, Klempner SJ, Allen JN, et al.&nbsp;Results and molecular correlates from a pilot study of neoadjuvant induction FOLFIRINOX followed by chemoradiation and surgery for gastroesophageal adenocarcinomas. Clin Cancer Res. 2021;27:6343–53.</p>
<p>dMMR and MsiHI - 5FU/platin ie inefficient and deleteriousReynolds JV, Preston SR, O’Neill B, et al.&nbsp;Trimodality therapy versus perioperative chemotherapy in the management of locally advanced adenocarcinoma of the oesophagus and oesophagogastric junction (Neo-AEGIS): an open-label, randomised, phase 3 trial. Lancet Gastroenterol Hepatol 2023; 8: 1015–27.</p>
<p>Optimal therapy for esophageal ad GE junction adenocarcinoma involves peroperative (“neoadjuvant”) chemoradiation or chemotherapy, followed by surgical resection.</p>
<p>Simple paradigm made more complicated:</p>
<ul>
<li><p>pCR rate of 23% suggests that if these patients could be identified, some patients could be spared surgery</p></li>
<li><p>CROSS trial: 30% of patients had cancer recurrence at 2 years</p>
<ul>
<li>If these patients oculd b identified earlier - no surgery or more neoadjuvant</li>
</ul></li>
</ul>
</section>
<section id="detection-of-residual-disease-after-chemoradiation" class="slide level2 smaller scrollable">
<h2>Detection of residual disease after chemoradiation</h2>
<p>Cologne: 80 patients with localy-advanced esophageal cancer were treated by cheoradiation followed by esophagectomy. Pathology of resection specimen was predictive of survival. To predict histopathologic response, EGD had a sensitivityof 605, Rebiopsy 39%, and EUS 7%. <span class="citation" data-cites="schneider248">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick=""><strong>schneider248?</strong></a>)</span></p>
<p>Reveiw of CT, EUS, PET for assessment of response to chemoRT <span class="citation" data-cites="westerterp842">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick=""><strong>westerterp842?</strong></a>)</span> Maximum oint values for sensitivity and specificity were 54% for CT, 86% for EUS, 85% for PET.</p>
<p>Munich: 40 patient with adenocarcinoma GE junction treated with trimodality therapy. Reduction in SUV was -54% in responders vs -15% in non-responders. pCT in 53% of metanbolic responders but in only 5% of non-responders. <span class="citation" data-cites="weber3058">(<a href="#/detection-of-residual-disease-after-chemoradiation" role="doc-biblioref" onclick="">Weber et al., 2001</a>)</span></p>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-azad494" class="csl-entry" role="listitem">
Azad, T.D., Chaudhuri, A.A., Fang, P., Qiao, Y., Esfahani, M.S., Chabon, J.J., Hamilton, E.G., Yang, Y.D., Lovejoy, A., Newman, A.M., Kurtz, D.M., Jin, M., Schroers-Martin, J., Stehr, H., Liu, C.L., Hui, A.B.-Y., Patel, V., Maru, D., Lin, S.H., Alizadeh, A.A., Diehn, M., 2020. Circulating <span>Tumor</span> <span>DNA</span> <span>Analysis</span> for <span>Detection</span> of <span>Minimal</span> <span>Residual</span> <span>Disease</span> <span>After</span> <span>Chemoradiotherapy</span> for <span>Localized</span> <span>Esophageal</span> <span>Cancer</span>. Gastroenterology 158, 494–505.e6. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/31711920">31711920</a>
</div>
<div id="ref-cabalage120" class="csl-entry" role="listitem">
Cabalag, C.S., Yates, M., Corrales, M.B., Yeh, P., Wong, S.Q., Zhang, B.Z., Fujihara, K.M., Chong, L., Hii, M.W., Dawson, S.-J., Phillips, W.A., Duong, C.P., Clemons, N.J., 2022. Potential <span>Clinical</span> <span>Utility</span> of a <span>Targeted</span> <span>Circulating</span> <span>Tumor</span> <span>DNA</span> <span>Assay</span> in <span>Esophageal</span> <span>Adenocarcinoma</span>. Annals of Surgery 276, e120–e126. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/35737908">35737908</a>
</div>
<div id="ref-egyud343" class="csl-entry" role="listitem">
Egyud, M., Tejani, M., Pennathur, A., Luketich, J., Sridhar, P., Yamada, E., Ståhlberg, A., Filges, S., Krzyzanowski, P., Jackson, J., Kalatskaya, I., Jiao, W., Nielsen, G., Zhou, Z., Litle, V., Stein, L., Godfrey, T., 2019. Detection of <span>Circulating</span> <span>Tumor</span> <span>DNA</span> in <span>Plasma</span>: <span>A</span> <span>Potential</span> <span>Biomarker</span> for <span>Esophageal</span> <span>Adenocarcinoma</span>. The Annals of Thoracic Surgery 108, 343–349. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/31059681">31059681</a>
</div>
<div id="ref-eyck35" class="csl-entry" role="listitem">
Eyck, B.M., Jansen, M.P., Noordman, B.J., Atmodimedjo, P.N., Wilk, B.J. van der, Martens, J.W., Helmijr, J.A., Beaufort, C.M., Mostert, B., Doukas, M., Wijnhoven, B.P., Lagarde, S.M., Lanschot, J.J.B. van, Dinjens, W.N., 2023. Detection of circulating tumour <span>DNA</span> after neoadjuvant chemoradiotherapy in patients with locally advanced oesophageal cancer. The Journal of Pathology 259, 35–45. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/36196486">36196486</a>
</div>
<div id="ref-huffmane2200420" class="csl-entry" role="listitem">
Huffman, B.M., Aushev, V.N., Budde, G.L., Chao, J., Dayyani, F., Hanna, D., Botta, G.P., Catenacci, D.V.T., Maron, S.B., Krinshpun, S., Sharma, S., George, G.V., Malhotra, M., Jurdi, A., Moshkevich, S., Aleshin, A., Kasi, P.M., Klempner, S.J., 2022. Analysis of <span>Circulating</span> <span>Tumor</span> <span>DNA</span> to <span>Predict</span> <span>Risk</span> of <span>Recurrence</span> in <span>Patients</span> <span>With</span> <span>Esophageal</span> and <span>Gastric</span> <span>Cancers</span>. JCO precision oncology 6, e2200420. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/36480779">36480779</a>
</div>
<div id="ref-iden83" class="csl-entry" role="listitem">
Iden, C.R., Mustafa, S.M., Øgaard, N., Henriksen, T., Jensen, S.Ø., Ahlborn, L.B., Egebjerg, K., Baeksgaard, L., Garbyal, R.S., Nedergaard, M.K., Achiam, M.P., Andersen, C.L., Mau-Sørensen, M., 2025. Circulating tumor <span>DNA</span> predicts recurrence and survival in patients with resectable gastric and gastroesophageal junction cancer. Gastric Cancer: Official Journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association 28, 83–95. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/39369091">39369091</a>
</div>
<div id="ref-kato6248" class="csl-entry" role="listitem">
Kato, S., Okamura, R., Baumgartner, J.M., Patel, H., Leichman, L., Kelly, K., Sicklick, J.K., Fanta, P.T., Lippman, S.M., Kurzrock, R., 2018. Analysis of <span>Circulating</span> <span>Tumor</span> <span>DNA</span> and <span>Clinical</span> <span>Correlates</span> in <span>Patients</span> with <span>Esophageal</span>, <span>Gastroesophageal</span> <span>Junction</span>, and <span>Gastric</span> <span>Adenocarcinoma</span>. Clinical Cancer Research: An Official Journal of the American Association for Cancer Research 24, 6248–6256. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/30348637">30348637</a>
</div>
<div id="ref-landere2400288" class="csl-entry" role="listitem">
Lander, E.M., Aushev, V.N., Huffman, B.M., Hanna, D., Dutta, P., Ferguson, J., Sharma, S., Jurdi, A., Liu, M.C., Eng, C., Klempner, S.J., Gibson, M.K., 2024. Circulating <span>Tumor</span> <span>DNA</span> as a <span>Prognostic</span> <span>Biomarker</span> for <span>Recurrence</span> in <span>Patients</span> <span>With</span> <span>Locoregional</span> <span>Esophagogastric</span> <span>Cancers</span> <span>With</span> a <span>Pathologic</span> <span>Complete</span> <span>Response</span>. JCO precision oncology 8, e2400288. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/39642325">39642325</a>
</div>
<div id="ref-weber3058" class="csl-entry" role="listitem">
Weber, W.A., Ott, K., Becker, K., Dittler, H.J., Helmberger, H., Avril, N.E., Meisetschläger, G., Busch, R., Siewert, J.R., Schwaiger, M., Fink, U., 2001. Prediction of response to preoperative chemotherapy in adenocarcinomas of the esophagogastric junction by metabolic imaging. Journal of Clinical Oncology: Official Journal of the American Society of Clinical Oncology 19, 3058–3065. https://vb3lk7eb4t.search.serialssolutions.com/?id=<a href="https://www.ncbi.nlm.nih.gov/pubmed/11408502">11408502</a>
</div>
</div>
</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<div class="footer footer-default">

</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="site_libs/revealjs/plugin/search/search.js"></script>
  <script src="site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: false,

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'none',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1050,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    <script id="quarto-html-after-body" type="application/javascript">
      window.document.addEventListener("DOMContentLoaded", function (event) {
        const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
        tabsets.forEach(function(tabset) {
          const tabby = new Tabby('#' + tabset.id);
        });
        const isCodeAnnotation = (el) => {
          for (const clz of el.classList) {
            if (clz.startsWith('code-annotation-')) {                     
              return true;
            }
          }
          return false;
        }
        const onCopySuccess = function(e) {
          // button target
          const button = e.trigger;
          // don't keep focus
          button.blur();
          // flash "checked"
          button.classList.add('code-copy-button-checked');
          var currentTitle = button.getAttribute("title");
          button.setAttribute("title", "Copied!");
          let tooltip;
          if (window.bootstrap) {
            button.setAttribute("data-bs-toggle", "tooltip");
            button.setAttribute("data-bs-placement", "left");
            button.setAttribute("data-bs-title", "Copied!");
            tooltip = new bootstrap.Tooltip(button, 
              { trigger: "manual", 
                customClass: "code-copy-button-tooltip",
                offset: [0, -8]});
            tooltip.show();    
          }
          setTimeout(function() {
            if (tooltip) {
              tooltip.hide();
              button.removeAttribute("data-bs-title");
              button.removeAttribute("data-bs-toggle");
              button.removeAttribute("data-bs-placement");
            }
            button.setAttribute("title", currentTitle);
            button.classList.remove('code-copy-button-checked');
          }, 1000);
          // clear code selection
          e.clearSelection();
        }
        const getTextToCopy = function(trigger) {
            const codeEl = trigger.previousElementSibling.cloneNode(true);
            for (const childEl of codeEl.children) {
              if (isCodeAnnotation(childEl)) {
                childEl.remove();
              }
            }
            return codeEl.innerText;
        }
        const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
          text: getTextToCopy
        });
        clipboard.on('success', onCopySuccess);
        if (window.document.getElementById('quarto-embedded-source-code-modal')) {
          const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
            text: getTextToCopy,
            container: window.document.getElementById('quarto-embedded-source-code-modal')
          });
          clipboardModal.on('success', onCopySuccess);
        }
          var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
          var mailtoRegex = new RegExp(/^mailto:/);
            var filterRegex = new RegExp("https:\/\/gisurgonc\.github\.io\/talks\/");
          var isInternal = (href) => {
              return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
          }
          // Inspect non-navigation links and adorn them if external
         var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
          for (var i=0; i<links.length; i++) {
            const link = links[i];
            if (!isInternal(link.href)) {
              // undo the damage that might have been done by quarto-nav.js in the case of
              // links that we want to consider external
              if (link.dataset.originalHref !== undefined) {
                link.href = link.dataset.originalHref;
              }
            }
          }
        function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
          const config = {
            allowHTML: true,
            maxWidth: 500,
            delay: 100,
            arrow: false,
            appendTo: function(el) {
                return el.closest('section.slide') || el.parentElement;
            },
            interactive: true,
            interactiveBorder: 10,
            theme: 'light-border',
            placement: 'bottom-start',
          };
          if (contentFn) {
            config.content = contentFn;
          }
          if (onTriggerFn) {
            config.onTrigger = onTriggerFn;
          }
          if (onUntriggerFn) {
            config.onUntrigger = onUntriggerFn;
          }
            config['offset'] = [0,0];
            config['maxWidth'] = 700;
          window.tippy(el, config); 
        }
        const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
        for (var i=0; i<noterefs.length; i++) {
          const ref = noterefs[i];
          tippyHover(ref, function() {
            // use id or data attribute instead here
            let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
            try { href = new URL(href).hash; } catch {}
            const id = href.replace(/^#\/?/, "");
            const note = window.document.getElementById(id);
            if (note) {
              return note.innerHTML;
            } else {
              return "";
            }
          });
        }
        const findCites = (el) => {
          const parentEl = el.parentElement;
          if (parentEl) {
            const cites = parentEl.dataset.cites;
            if (cites) {
              return {
                el,
                cites: cites.split(' ')
              };
            } else {
              return findCites(el.parentElement)
            }
          } else {
            return undefined;
          }
        };
        var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
        for (var i=0; i<bibliorefs.length; i++) {
          const ref = bibliorefs[i];
          const citeInfo = findCites(ref);
          if (citeInfo) {
            tippyHover(citeInfo.el, function() {
              var popup = window.document.createElement('div');
              citeInfo.cites.forEach(function(cite) {
                var citeDiv = window.document.createElement('div');
                citeDiv.classList.add('hanging-indent');
                citeDiv.classList.add('csl-entry');
                var biblioDiv = window.document.getElementById('ref-' + cite);
                if (biblioDiv) {
                  citeDiv.innerHTML = biblioDiv.innerHTML;
                }
                popup.appendChild(citeDiv);
              });
              return popup.innerHTML;
            });
          }
        }
      });
      </script>
    

</body></html>