---
title: "ctDNA in Esophageal Cancer"
editor: visual
bibliography: zotero.bib
csl: elsevier-stanford.csl
link-citations: true
---

**Rotterdam** [@eyck35] Detection of circulating tumour DNA after neoadjuvant chemoradiotherapy in patients with locally advanced oesophageal cancer. J Pathol 2023; 259: 35–45. Samples taking before, 6 weeks after and 23 weeks after CRT. 29/31 patients had mutations detected on *tissue biopsy*. 24/31 patients had mutations detectable in cfDNA. Prei-itreatment cdTNA detected in 9 patients, four of whom had incurable disease progression before surgery (2 prior to ChemoRT and 2 prior to surgery). None of these 9 had a pCR. 6/9 had major residual disease after chemoRT. After ChemoRT, ctDNA was detected in 3 patients, two of whom had disease progression. Patients were examined wtih CT and EGD biopsies after ChemoRT as part of the SANO trial. ctDNA did not detect any additional patients with residual disease after ChemoRT. ctDNA analyzed with [Thermo Fisher Colon Oncoline kit](https://www.thermofisher.com/us/en/home/life-science/cancer-research/cancer-genomics/liquid-biopsy-cancer-research-applications/oncomine-cell-free-dna-assays-liquid-biopsy-clinical-research.html). $\Rightarrow$. ctDNA can detect some patients with high-risk disease who are at risk for progression (and should have further staging investigations prior to surgery OR be offered watch/wait), but did not help detect patients after ChemoRT who had residual disease (who would be candidates for salvage esophagectomy)

**Rotterdam** Restaging after chemoradiation. 

Melbourne [@cabalage120] 62 patients with adenocarcinoma with **tumor-informed** approach. 9 genes analyzed with targeted amplicon sequencing. Pre-treatment biopsies showed mutations in 55/62 (63%). ctDNA at baseline in 20/55 (36%).  The majority (90%) of patients with detectable ctDNA had either locally advanced tumors, nodal involvement or metastatic disease. In patients with locally advanced tumors, disease free survival (DFS) was more accurately stratified using pre-treatment ctDNA status \[HR 4.34 (95% CI 0.93-20.21); P = 0.05\] compared to nodal status on PET-CT. In an exploratory subgroup analysis, patients who are node negative but ctDNA positive have inferior DFS \[HR 11.71 (95% CI 1.16-118.80) P = 0.04\]. In blood samples taken before and following NAT, clearance of ctDNA after NAT was associated with a favourable response to treatment. Furthermore, patients who are ctDNA positive during post-treatment surveillance are at high risk of relapse.

San Diego: [@kato6248] 55 GE adenocarcinoma (mostly advanced disease; 9, surgically resectable) ctDNA with Guardant (49/55) or Foundation. Esophagus =11, GEJ = 17, gastric = 27. Seventy-six percent of patients (42/55) had ≥1 genomic alteration \[including variants of unknown significance (VUS)\] and 69.1% (38/55) had ≥1 characterized alteration (excluding VUSs). The median number of alterations per patient was 2 (range, 0–15). TP53 (50.9%, 28/55), PIK3CA (16.4%, 9/55), ERBB2 (14.5%, 8/55), and KRAS (14.5%, 8/55) genes were most frequently affected characterized alterations. Thirty-one patients also had tissue NGS. Concordance between tissue and ctDNA ranged from 61.3% (TP53 alterations) to 87.1% (KRAS alterations). ERBB2 alterations were significantly associated with poor overall survival (HR, 14.06; 95% confidence interval, 2.44–81.03; P = 0.003 multivariate analysis). Among patients with ≥1 alteration, no 2 patients had identical molecular portfolios.

Stanford: 45 patients receving chemoradiation for esophageal cancer. ctDNA analysis with deep squencing. Tumor-derived DNA represented 0.07% of all cell-free DNA. Detection of ctDNA following chemoradiotherapy was associated with tumor progression (hazard ratio, 18.7; P\<.0001), formation of distant metastases (hazard ratio, 32.1; P\<.0001), and shorter disease-specific survival times (hazard ratio, 23.1; P\<. 0001). A higher proportion of patients with tumor progression had new mutations detected in plasma samples collected after chemoradiotherapy than patients without progression (P=.03). Detection of ctDNA after chemoradiotherapy preceded radiographic evidence of tumor progression by an average of 2.8 months. Among patients who received chemoradiotherapy without surgery, combined ctDNA and metabolic imaging analysis predicted progression in 100% of patients with tumor progression, compared with 71% for only ctDNA detection and 57% for only metabolic imaging analysis


Zaanan A, Didelot A, Broudin C, et al. Longitudinal circulating tumor DNA (ctDNA) analysis during treatment (Tx) of locally advanced resectable (LAR) gastric or gastroesophageal junction (G/GEJ) adenocarcinoma (ADENOCA): the PLAGAST prospective biomarker study. J Clin Oncol 2024; 42: 4028 (abstract).

Elevated ctDNA baseline concentrations have also been reported to predict major residual disease after neoadjuvant chemoradiotherapy and disease progression before surgery.67,68 Several studies also identified ctDNA as a potential biomarker of recurrence and overall survival.67,69 In the ancillary study of the CRITICS trial (NCT00407186), Leal and colleagues identified post-neoadjuvant therapy or preoperative ctDNA as a predictive biomarker for pathological response (according to Mandard’s tumour regression grading \[TRG\] system0,70 with ctDNA analyses being consistent with TRG in 30 (70%) of 43 patients (p=0·03; Fisher’s exact test).71 Huffman and colleagues reported the largest cohort of patients with oesophagogastric cancers monitored with ctDNA.72 In their study, a personalised, multiplex PCRbased next-generation sequencing assay was used to retrospectively analyse ctDNA in 295 patients with oesophagogastric cancers presenting with different histology (squamous cell carcinomas and adenocarcinomas) profiles and spanning various disease stages (I–IV). In this cohort, 125 patients with stage I–III oesophagogastric cancers were analysed at any timepoint postoperatively (regardless of adjuvant treatment). The median recurrence-free survival was not reached in patients negative for ctDNA, whereas it was 9·6 months in patients positive for ctDNA (HR 23·6, 95% CI 10·2–66·0; p=0·0001).72 These results were observed irrespective of the anatomical localisation of the tumour. Recurrence rates were 5% in patients negative for ctDNA and 88% in patients positive for ctDNA. The patient population enrolled in this study was heterogeneous and further prospective studies are required. In the initial analysis of the PLAGAST study73 presented at the ASCO 2024 meeting, postoperative, tumour-informed ctDNA persistence was related to recurrence and overall survival. Additionally, the clearance of ctDNA after medical treatment could serve as a predictor of outcomes. To date, there are no available data for localised oesophagogastric adenocarcinoma, but in the metastatic setting, Kelly and colleagues observed in a phase 1–2 study a statistical correlation between mutation variant allele frequencies and overall response rate to ICIs in metastatic oesophagogastric adenocarcinoma.74 Patients with a decrease in variant allele frequencies at week 9 had a longer median OS (3.8 months, 95% CI 7·0–NR) than those with an increase in variant allele frequencies (8.2 months, 6·2–NR; HR 0·07, 95% CI 0·01–0·66).74 However, false-negative ctDNA results have been reported in patients with peritoneal carcinomatosis, restricting the utility of ctDNA as the sole predictive biomarker for oesophagogastric adenocarcinoma.75

68 Egyud M, Tejani M, Pennathur A, et al. Detection of circulating tumor DNA in plasma: a potential biomarker for esophageal adenocarcinoma. Ann Thorac Surg 2019; 108: 343–49.

69 Azad TD, Chaudhuri AA, Fang P, et al. Circulating tumor DNA analysis for detection of minimal residual disease after chemoradiotherapy for localized esophageal cancer. Gastroenterology 2020; 158: 494–505.


Chidambaram S, Markar SR. Clinical utility and applicability of circulating tumor DNA testing in esophageal cancer: a systematic review and meta-analysis. Dis Esophagus. 2022;35:doab046.

CRITICS trial [@leal525]

71 Leal A, van Grieken NCT, Palsgrove DN, et al. White blood cell and cell-free DNA analyses for detection of residual disease in gastric cancer. Nat Commun 2020; 11: 525.

[@huffmane2200420] ctDNA levels were analyzed in the preoperative, postoperative, and surveillance settings in patients with EGC using a **personalized multiplex polymerase chain reaction-based next-generation sequencing assay**. Plasma samples (n = 943) from 295 patients at \> 70 institutions were collected before surgery, postoperatively, and/or serially during routine clinical follow-up from September 19, 2019, to February 21, 2022. A total of 295 patients with EGC were analyzed, and 212 patients with stages I-III disease were further explored. Pretreatment ctDNA was detected in 96% (23/24) of patients with preoperative time points. Postoperative ctDNA was detected in 23.5% (16/68) of patients with stage I-III EGC within 16 weeks (molecular residual disease window) after surgery without receiving systemic therapy. ctDNA detection at any time point after surgery (hazard ratio \[HR\], 23.6; 95% CI, 10.2 to 66.0; *P* \< .0001), within the molecular residual disease window (HR, 10.7; 95% CI, 4.3 to 29.3; *P* \< .0001), and during the surveillance period (HR, 17.7; 95% CI, 7.3 to 50.7; *P* \< .0001) was associated with shorter recurrence-free survival. In multivariable analysis, ctDNA status and clinical stage of disease were independently associated with outcomes.

Copenhagen [@iden83]: Patients with resectable gastric and GEJ AC, undergoing perioperative chemotherapy and surgery, were prospectively enrolled. Serial plasma samples were collected at baseline, after one cycle of chemotherapy, after preoperative chemotherapy, and after surgery. ctDNA was assessed by a **ddPCR test (TriMeth),** which targets the gastrointestinal cancer-specific methylation patterns of the genes C9orf50, KCNQ5, and CLIP4.

**Results:** ctDNA analysis was performed on 229 plasma samples from 86 patients. At baseline, ctDNA was detected in 56% of patients, which decreased to 37% following one cycle of chemotherapy, 25% after preoperative chemotherapy and 15% after surgical resection. The presence of ctDNA after one cycle of chemotherapy was associated with reduced recurrence-free survival (RFS) (HR = 2.54, 95% confidence interval (CI) 1.33-4.85, p = 0.005) and overall survival (OS) (HR = 2.23, 95% CI 1.07-4.62, p = 0.032). Similarly, ctDNA after surgery was associated with significantly shorter RFS (HR = 6.22, 95% CI 2.39-16.2, p \< 0.001) and OS (HR = 6.37, 95% CI 2.10-19.3, p = 0.001). Multivariable regression analysis confirmed ctDNA after surgery as an independent prognostic factor (p \< 0.001).


Wo JY, Clark JW, Eyler CE, Mino-Kenudson M, Klempner SJ, Allen JN, et al. Results and molecular correlates from a pilot study of neoadjuvant induction FOLFIRINOX followed by chemoradiation and surgery for gastroesophageal adenocarcinomas. Clin Cancer Res. 2021;27:6343–53.



PET alters management in 16% of cases of gastric cancer [@smyth5481]

ESMO guidlines recommend PET for esophagus but not gastric

dMMR and MsiHI - 5FU/platin ie inefficient and deleteriousReynolds JV, Preston SR, O’Neill B, et al. Trimodality therapy versus perioperative chemotherapy in the management of locally advanced adenocarcinoma of the oesophagus and oesophagogastric junction (Neo-AEGIS): an open-label, randomised, phase 3 trial. Lancet Gastroenterol Hepatol 2023; 8: 1015–27.

But neoadjuvant FLOT indocued 27% pCR in the DNATE study [@lorenzen410] Lorenzen S, Götze TO, Thuss-Patience P, et al. Perioperative atezolizumab plus fluorouracil, leucovorin, oxaliplatin, and docetaxel for resectable esophagogastric cancer: interim results from the randomized, multicenter, phase II/III DANTE/IKF-s633 Trial. J Clin Oncol 2024; 42: 410–20.

an T, Zhang X-F, Liang C, Liao C-W, Li J-Y, Zhou Y-M. The prognostic value of a pathologic complete response after neoadjuvant therapy for digestive cancer: systematic review and meta-analysis of 21 studies. Ann Surg Oncol 2019; 26: 1412–20.

Bite on Bite Biopsies (PreSane) Noordman BJ, Spaander MCW, Valkema R, et al. Detection of residual disease after neoadjuvant chemoradiotherapy for oesophageal cancer (preSANO): a prospective multicentre, diagnostic cohort study. Lancet Oncol 2018; 19: 965–74.

Noordman BJ, Wijnhoven BPL, Lagarde SM, et al. Active surveillance in clinically complete responders after neoadjuvant chemoradiotherapy for esophageal or junctional cancer. Dis Esophagus 2017; 30: 1–8.

Optimal therapy for esophageal ad GE junction adenocarcinoma involves peroperative ("neoadjuvant") chemoradiation or chemotherapy, followed by surgical resection.

Simple paradigm made more complicated:

-   pCR rate of 23% suggests that if these patients could be identified, some patients could be spared surgery

-   CROSS trial: 30% of patients had cancer recurrence at 2 years

    -   If these patients oculd b identified earlier - no surgery or more neoadjuvant

| Genes   | Col2          | Stanford   | Rotterdam | Melbourne      | Col6 |
|---------|---------------|------------|-----------|----------------|------|
|         | [@hofste4417] | [@azad494] | [@eyck35] | [@cabalage120] |      |
| APC     | x             |            | X         | X              |      |
| ARID1A  | x             |            |           | X              |      |
| AKT1    |               |            | X         |                |      |
| BRAF    | x             |            | X         |                |      |
| CDKN2A  | x             | Common     |           | X              |      |
| CTNNB1  | x             |            | X         |                |      |
| EGFR    |               |            | X         |                |      |
| ERBB2   | x             | Common     | X         |                |      |
| FBXW7   | x             |            | X         |                |      |
| GNAS    | x             |            | X         |                |      |
| KRAS    | x             |            | X         | X              |      |
| MAP2K1  |               |            | X         |                |      |
| NRAS    |               |            | X         |                |      |
| NRG1    |               |            |           | X              |      |
| PIK3CA  | x             |            | X         | X              |      |
| RNF43   | x             |            |           |                |      |
| SMAD4   | x             |            | X         | X              |      |
| SMARCA4 |               |            |           | X              |      |
| TGFB2   | x             |            |           |                |      |
| TP53    | x             | Common     | X         | X              |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
|         |               |            |           |                |      |
